lib/src/screens/exchange/exchange_template_page.dart

```diff
@@ -190,8 +189,8 @@ class ExchangeTemplatePage extends BasePage {
                                 .decorationColor,
                             currencyValueValidator: AmountValidator(
                                 type: exchangeViewModel.wallet.type),
-                            addressTextFieldValidator: AddressValidator(
-                                type: exchangeViewModel.receiveCurrency),
+                            //addressTextFieldValidator: AddressValidator(
```

Comment:
Better to make new validator which will use address validator and domain validator (create if not exists) and keep validation for address fields.